file-server/app.py

```diff
@@ -0,0 +1,149 @@
+import os
+import json
+from dotenv import load_dotenv
+from flask import Flask, request, jsonify
+from flask_cors import CORS
+from imgget import search_images, download_images
+import voiceovergen as vo
+import moviegen as mg
+import llm_req as llm
+import userstuff as us
+
+load_dotenv()
+
+PORT = os.environ["FILE_SERVER_PORT"]
+app = Flask(__name__)
+CORS(app)
+PALM_API_KEY = os.environ["PALMAPI"]
+
+
+@app.route("/text", methods=["POST"])
+def text():
+    """Send prompt to palm 2 API, get text. Use keywords to get images."""
+    if request.method == "POST":
```

Comment:
Is the request method not already set here:

`@app.route("/text", methods=["POST"])`